src/include/thread_group.h

```diff
@@ -22,11 +22,12 @@ struct __wt_thread {
  * groups.
  */
 /* AUTOMATIC FLAG VALUE GENERATION START */
-#define WT_THREAD_ACTIVE 0x1u     /* Thread is active or paused */
-#define WT_THREAD_CAN_WAIT 0x2u   /* WT_SESSION_CAN_WAIT */
-#define WT_THREAD_PANIC_FAIL 0x4u /* Panic if the thread fails */
-#define WT_THREAD_RUN 0x8u        /* Thread is running */
-                                  /* AUTOMATIC FLAG VALUE GENERATION STOP */
+#define WT_THREAD_ACTIVE 0x01u   /* Thread is active or paused */
+#define WT_THREAD_CAN_WAIT 0x02u /* WT_SESSION_CAN_WAIT */
+#define WT_THREAD_ISOLATION_READ_COMMITTED 0x04u
```

Comment:
I think it is odd to specify isolation level for thread. That is the concept exists at session level. Can we call this WT_THREAD_EVICTION?

Reply:
The idea here is to force an isolation level at the time of creating sessions for the new thread. I intentionally wanted to make the name generic so other parts of the code can also use this flag to create threads (and associated sessions) with specific isolation level.

Reply:
I'm with Haseeb. A session is a thread context, so I think it's reasonable to map the expected isolation level to the flag here.